packages/styles/section-header.css

```diff
@@ -0,0 +1,54 @@
+:root {
+  --section-header-description-color: var(--gray-60);
+  --section-header-heading-color: var(--gray-90);
```

Comment:
I don't think we need component level overrides, we already have this being set in `base.css`:

https://github.com/dequelabs/cauldron/blob/23bcd75d0d038d2816125c14e34cfa15b54f468f/packages/styles/base.css#L45-L54

```suggestion
```